### PR TITLE
gnome-characters: update to 46.0.

### PR DESCRIPTION
**Pull request by fanyx**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/gnome-characters/template [hunk 1]
```diff
@@ -13,7 +13,8 @@ short_desc="Utility to find and insert unusual characters for GNOME"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Design/Apps/CharacterMap"
+changelog="https://gitlab.gnome.org/GNOME/gnome-characters/-/raw/main/NEWS"
```
Comment (oreo639) [inline srcpkgs/gnome-characters/template:16]: This should be `https://gitlab.gnome.org/GNOME/gnome-characters/-/raw/gnome-46/NEWS`

**Comment by fanyx** [comment]
still seeing the test error regardless of where i add `mesa-dri`

**Comment by oreo639** [comment]
Sorry, yeah, the crash is in cairo due to no font being installed.

**Comment by oreo639** [comment]
> any mock fonts available or do i just throw in `noto-font-ttf`?

For gnome stuff, it makes sense to use `cantarell-fonts`, but in this case, it is definitely a bug, so don't add it.

**Comment by fanyx** [comment]
Builds work fine on my host machines btw
i've only seen this issue when no display is available like in the pipeline or using WSL

**Comment by oreo639** [comment]
The crashing got resolved tangentially in gtk4 4.14.2

**Pull request merged**